packages/superset-ui-core/test/translation/Translator.test.ts

```diff
@@ -34,6 +35,18 @@ configure({
 });
 
 describe('Translator', () => {
+  const spy = jest.spyOn(logging, 'warn');
+
+  beforeAll(() => {
+    spy.mockImplementation((info: any) => {
```

Comment:
Can you explain why do we need to mock logging methods? And why do they throw errors?

Reply:
Oh, I see, so we just bring back relevant changes from removed library 👍

Reply:
There are some test cases that need to get warning message.